src/datasets/other/newtab/reference.md

```diff
@@ -0,0 +1,83 @@
+# Newtab Interactions
```

Comment:
There are a few missing additions to the `.spelling` file: https://github.com/mozilla/data-docs/actions/runs/4086382290/jobs/7045605705

Reply:
Thanks, I will update this.